runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -934,43 +953,25 @@ NOTE: Completion works for many commands. It is especially useful for `:help`{vi
 
  4. Type `:q`{vim} to close the help window
 
- 5. Create a vimrc startup script to keep your preferred settings.
+ 5. Create a init.vim startup script to keep your preferred settings.
 
  6. While in command mode, press `<C-d>`{normal} to see possible completions.
     Press `<Tab>`{normal} to use one completion.
 
 # CONCLUSION
 
-This was intended to give a brief overview of the Vim editor, just enough to
-allow you to use the editor fairly easily. It is far from complete as Vim has
+This was intended to give a brief overview of the Neovim editor, just enough to
+allow you to use it fairly easily. It is far from complete as Neovim has
 many many more commands. Consult the help often.
-
-There are many resources online to learn more about vim. Here's a bunch of
-them:
-
-- *Learn Vim Progressively*:
-  http://yannesposito.com/Scratch/en/blog/Learn-Vim-Progressively/
-- *Learning Vim in 2013*:
-  http://benmccormick.org/learning-vim-in-2014/
-- *Vimcasts*:
-  http://vimcasts.org/
-- *Vim Video-Tutorials by Derek Wyatt*:
-  http://derekwyatt.org/vim/tutorials/
-- *Learn Vimscript the Hard Way*:
-  http://learnvimscriptthehardway.stevelosh.com/
-- *7 Habits of Effective Text Editing*:
-  http://www.moolenaar.net/habits.html
-- *vim-galore*:
-  https://github.com/mhinz/vim-galore
-
-If you prefer a book, *Practical Vim* by Drew Neil is recommended often
-(the sequel, *Modern Vim*, includes material specific to nvim).
+There are also countless great tutorials and videos to be found online, some
+even by the maintainers of Neovim.
```

Comment:
Why did you remove the references? Is the book really not applicable?

In general, I think it's fine to also recommend Vim resources to very beginners, since the fundamentals are not different. Maybe leave them in but preface them with an explanation? ("For the basics, resources on learning Vim are also helpful since the fundamental editing model is the same, but be aware that some options may be set differently by default. Here's a partial list.")

Feel free to remove resources that are too stale, though. I wouldn't stress the "maintainers of Neovim", either.

Reply:
I was just making the assumption most people don't bother with these and jump straight to YouTube or Google. I shall update how you suggested

Reply:
Yeah, which drops them into a sea of bad/outdated/monetized advice ;) That's exactly why a curated list is helpful.